### ISSUE TITLE
[Feature Request]  Notification period and import

### ISSUE THREAD

**Issue by frgalan** [ellite/Wallos#230]
Hi

First congratulations for the project, amazing, something that I was looking for a long time.

I think would be great if for each subscription we could select between different notification profiles (1 day, 1 week, 1 month) or select an individual notification period for each one.

Also could be great to have the option to import CSV from different options (trackmysubs etc)

More suggestions, instead of enabling notifications for each subscription have this by default and give the option to remove notifications. Have a bigger note area.

**Comment by programmer24601** [comment]
It'd be nice to not be limited to a max advance notification of 90 days. For example, I'm wanting to add drivers license and passport renewals (as they are effectively subscriptions), but (at least in my country) passports need renewing every 10 years, and that process needs to start 6 months before the expiry date. 

I could fudge it with a 9.5 year renewal frequency, but I'd rather not have to.

In summary, down with arbitrary limits!
